--- a/include/git2/repository.h
+++ b/include/git2/repository.h
@@ -90,11 +90,7 @@ GIT_EXTERN(int) git_repository_open(git_repository **repository, const char *pat
  *
  * @return 0 on success; error code otherwise
  */
-GIT_EXTERN(int) git_repository_open2(git_repository **repository,
-		const char *git_dir,
-		const char *git_object_directory,
-		const char *git_index_file,
-		const char *git_work_tree);
+GIT_EXTERN(int) git_repository_open2(git_repository **repository, const char *git_dir, const char *git_object_directory, const char *git_index_file, const char *git_work_tree);
 
 
 /**
@@ -126,11 +122,7 @@ GIT_EXTERN(int) git_repository_open2(git_repository **repository,
  * @return 0 on success; error code otherwise
  */
 
-GIT_EXTERN(int) git_repository_open3(git_repository **repository,
-		const char *git_dir,
-		git_odb *object_database,
-		const char *git_index_file,
-		const char *git_work_tree);
+GIT_EXTERN(int) git_repository_open3(git_repository **repository, const char *git_dir, git_odb *object_database, const char *git_index_file, const char *git_work_tree);
 
 
 /**
